content/doc/book/pipeline/syntax.adoc

```diff
@@ -584,6 +584,9 @@ time at which the line was emitted. For example: `options { timestamps() }`
 parallelsAlwaysFailFast:: Set failfast true for all subsequent parallel stages in the pipeline.
 For example: `options { parallelsAlwaysFailFast() }`
 
+checkoutExtensions:: Applies extentions to all SCM checkouts in the pipeline. Currently only CleanCheckout is available.
```

Comment:
Has this been implemented?  The [pull request](https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/307) seems to have been closed without being merged.  Is the change made somewhere else

Minor typographical correction:

```suggestion
checkoutExtensions:: Applies extensions to all SCM checkouts in the pipeline. Currently only CleanCheckout is available.
```